SamsungPay/src/main/AndroidManifest.xml

```diff
@@ -2,4 +2,12 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.braintreepayments.api.samsungpay">
 
+    <application>
+        <meta-data android:name="spay_sdk_api_level" android:value="2.4" />
```

Comment:
What does the 2.4 mean here? It looks like we use version 2.5 of the Samsung Pay SDK.

Reply:
I'm not entirely sure. There's a bit of a description [here](https://d3sfvyfh4b9elq.cloudfront.net/ptr/guide/techguide/v1.4/guide/05_00_understanding_debug_api_key_api_level.html). I think it has something to do with backward compatability with older versions of the SamsungPay app? I kept `2.4` because it's what we use in the current implementation.

Reply:
Ah ok that seems good to me then 👍